nohup: ignoring input
cuda available: False 
cuda available: False 
cuda available: False 
cuda available: False 
model initialized
training func is running after this line
Supervised Validation started
heuristic: one_step_greedy learning rate: 0.0001 epochs: 21 DataSet: UFSMDataset

/local_workspace/liudefen/PycharmProjects/PycharmProjects/GraphRL/gcn/models_gcn.py:98: UserWarning: Implicit dimension choice for log_softmax has been deprecated. Change the call to include dim=X as an argument.
  features = F.log_softmax(features.t())
epochs 0 loss 0.030830584075772275 one_step_greedy_performance 2502.6187050359713 gcn_performance 3589.6978417266187 random_performance 13670.212230215828
epochs 1 loss 0.004830328289504199 one_step_greedy_performance 2502.6187050359713 gcn_performance 13363.428057553956 random_performance 13670.212230215828
epochs 2 loss 0.009848533988946109 one_step_greedy_performance 2502.6187050359713 gcn_performance 3253.028776978417 random_performance 13670.212230215828
epochs 3 loss 0.011322812352726861 one_step_greedy_performance 2502.6187050359713 gcn_performance 3101.63309352518 random_performance 13670.212230215828
epochs 4 loss 0.011587883651473573 one_step_greedy_performance 2502.6187050359713 gcn_performance 3184.661870503597 random_performance 13670.212230215828
epochs 5 loss 0.012073464775845652 one_step_greedy_performance 2502.6187050359713 gcn_performance 3173.3021582733813 random_performance 13670.212230215828
epochs 6 loss 0.01136345735281862 one_step_greedy_performance 2502.6187050359713 gcn_performance 3739.061151079137 random_performance 13670.212230215828
epochs 7 loss 0.012103998366459472 one_step_greedy_performance 2502.6187050359713 gcn_performance 2957.838129496403 random_performance 13670.212230215828
epochs 8 loss 0.011787708804984743 one_step_greedy_performance 2502.6187050359713 gcn_performance 3017.776978417266 random_performance 13670.212230215828
epochs 9 loss 0.011791009527369781 one_step_greedy_performance 2502.6187050359713 gcn_performance 3112.7589928057555 random_performance 13670.212230215828
epochs 10 loss 0.012061174988839721 one_step_greedy_performance 2502.6187050359713 gcn_performance 3533.26618705036 random_performance 13670.212230215828
epochs 11 loss 0.011660655606789111 one_step_greedy_performance 2502.6187050359713 gcn_performance 3262.046762589928 random_performance 13670.212230215828
epochs 12 loss 0.011715454125359594 one_step_greedy_performance 2502.6187050359713 gcn_performance 3034.388489208633 random_performance 13670.212230215828
epochs 13 loss 0.011387901932673966 one_step_greedy_performance 2502.6187050359713 gcn_performance 3530.143884892086 random_performance 13670.212230215828
epochs 14 loss 0.011807312846183316 one_step_greedy_performance 2502.6187050359713 gcn_performance 2938.6510791366904 random_performance 13670.212230215828
Traceback (most recent call last):
  File "test_supervised_validation.py", line 146, in <module>
    prune = True
  File "/local_workspace/liudefen/PycharmProjects/PycharmProjects/GraphRL/supervised/train_supervised_learning.py", line 798, in validation_epochs
    + '_epochs_' + str(epoch) + '_cuda.pth', map_location='cpu'))
  File "/home/x86_64-unknown-linux_ol7-gnu/anaconda-5.2.0/envs/pytorch/lib/python3.6/site-packages/torch/serialization.py", line 301, in load
    f = open(f, 'rb')
FileNotFoundError: [Errno 2] No such file or directory: './supervised/models/one_step_greedy/SmallErgTraining/layers3_test_lr0.0001/per_epochs/gcn_policy_one_step_greedy_pre_ErgDataset_epochs_15_cuda.pth'
